prepareRelease.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+declare nextVersion=$1
+
+sed -i.bak -e "s/.setVersion('.*')/.setVersion('$nextVersion')/g" src/main.ts
+rm src/main.ts.bak
+
+SWAGGER_VERSION=$nextVersion POLYMESH_NODE_URL='wss://testnet-rpc.polymesh.live' yarn generate:swagger > /dev/null 2>&1
```

Comment:
I have stuck with testnet RPC for now. We can later think of replace this with a local chain @polymath-eric